cmake/linux/installer

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+set -e
+[ "$1" ]
+
+VERSION="$1"
+DIR="$HOME"/.lmms-$VERSION
```

Comment:
@tres I'm  all for a self contained ~home directory install. This is usually how I will install most software that I build myself. I guess most users will not be running a machine with multiple users, therefore a system wide install is not needed anyway.